source/administration/telemetry.rst

```diff
@@ -162,24 +160,28 @@ The following list details the types of Incident Management metadata we collect:
 - ``NumChecklists``: Number of stages in this incident.
 - ``TotalChecklistItems``: Number of tasks in this incident.
 - ``ActiveStage``: A number indicating the stage of the incident (0-based).
-- ``Public``: ``true`` if the incident was public, ``false`` if it was private.
+- ``Action``: The type of action performed against the incident: ``create``, ``end``, ``restart``, ``change_stage``, ``change_commander``, ``update_status``.
+- ``Public``: When creating an incident, ``true`` if it is public, and ``false`` if it is private.
 
 **Data collected in tasks events**
 
-- ``incidentID``: Unique identifier of the incident.
+- ``IncidentID``: Unique identifier of the incident.
 - ``NewState``: ``null`` if the task is uncompleted, ``done`` if the task was marked completed.
-- ``WasCommander``: ``true`` if the userId who initiated the event was also the commander of the event, ``false`` if not.
-- ``WasAssignee``: ``true`` if the userId who initiated the event was also the assignee of the event, ``false`` if not.
+- ``Action``: The type of action performed against the task: ``add_task``, ``remove_task``, ``rename_task``, ``modify_task_state``, ``move_task``, ``set_assignee_for_task``, ``run_task_slash_command``.
+- ``NewState``: When modifying a task state, ``true`` if the task is now checked, ``false`` if the task is now unchecked.
+- ``WasCommander``: When modifying a task state, ``true`` if the userId who initiated the event was also the commander of the event, and ``false`` otherwise.
+- ``WasAssignee``: When modifying a task state, ``true`` if the userId who initiated the event was also the assignee of the event, ``false`` otherwise.
```

Comment:
```suggestion
- ``WasAssignee``: When modifying a task state, ``true`` if the userId who initiated the event was also the assignee of the event, and ``false`` otherwise.
```